pvlib/pvsystem.py

```diff
@@ -164,15 +163,14 @@ class PVSystem:
 
     inverter : string, optional
         The model name of the inverters.
-        May be used to look up the inverter_parameters dictionary
-        via some other method.
 
     inverter_parameters : dict or Series, optional
         Inverter parameters as defined by the SAPM, CEC, or other.
 
-    racking_model : string, default 'open_rack'
-        Valid strings are 'open_rack', 'close_mount', and 'insulated_back'.
-        Used to identify a parameter set for the SAPM cell temperature model.
+    racking_model : string, optional
+        Valid strings are 'open_rack', 'close_mount', 'freestanding', or
+        'insulated_back'.
+        Used to identify a parameter set for the cell temperature model.
```

Comment:
Isn't `insulated` another valid option?

Reply:
Right, in the pvsyst cell temperature model I think `insulated` is the parameter equivalent to `insulated_back` in the sapm cell temperature model.
(if I have got that right) is there a reason we have these two names for the same thing?
I have added it as an option in the last commit btw.

Reply:
It is.